### PR TITLE
Fix Python3 compatibility for lib/galaxy/web/

### PR DESCRIPTION
**Pull request by nsoranzo**
xref. #1715

### CODE REVIEW

File: lib/galaxy/web/stack/__init__.py [hunk 1]
```diff
@@ -370,8 +370,8 @@ def _do_uwsgi_postfork():
 
 
 def _mule_fixup():
-    import urllib2
-    urllib2._opener = None
+    from six.moves.urllib.request import install_opener
+    install_opener(None)
```
Comment (nsoranzo) [inline lib/galaxy/web/stack/__init__.py:374]: @natefoo This should be equivalent, does it look good to you?

Comment (dannon) [reply lib/galaxy/web/stack/__init__.py:374]: I tested a few things locally and it does look like it's equivalent and should make sure that the `_opener` is None (which it seems to default to, but I'm assuming this is intended to override something uwsgi is setting), to me:

```
In [1]: import urllib2

In [2]: urllib2._opener

In [3]: from six.moves.urllib.request import install_opener

In [4]: install_opener("foo")

In [5]: urllib2._opener
Out[5]: 'foo'

In [6]: install_opener(None)

In [7]: urllib2._opener

```

Comment (dannon) [reply lib/galaxy/web/stack/__init__.py:374]: @nsoranzo Did you want to wait for more feedback, or is this ready to merge in your opinion?  It looks good to me.

Comment (nsoranzo) [reply lib/galaxy/web/stack/__init__.py:374]: I think it's ready to merge, I also checked the Python2 and Python3 implementations and these are equivalent.

Comment (dannon) [reply lib/galaxy/web/stack/__init__.py:374]: @nsoranzo Cool!

**Pull request merged**